src/settings.js

```diff
@@ -681,9 +681,26 @@ var FILESYSTEM = 1;
 // some JS that does, you might need this.
 var FORCE_FILESYSTEM = 0;
 
-// This mode is intended for use with Node.js (and will throw if the build runs
-// in other engines).  The File System API will directly use Node.js API without
-// requiring `FS.mount()`.  The initial working directory will be same as
+// Enables support for the in-memory filesystem backend. This is the only
+// one that is enabled by default.
+var MEMFS = 1;
+
+// Enables support for the IDBFS filesystem backend.
+var IDBFS = 0;
+
+// Enables support for the Node.js filesystem backend.
+var NODEFS = 0;
```

Comment:
Is this different to existing NODERAWFS setting?

Reply:
Yes, NODEFS and NODERAWFS are separate settings. NODERAWFS not only adds support for Node.js filesystem access, but also replaces everything else with it, so everything goes straight to node. We use it for configure checks for example, where sometimes they need to read a file.